array-manipulation-1/src/initial.js

```diff
@@ -0,0 +1,11 @@
+/* exported initial */
+
+function initial(array) {
+  const newArray = [];
+  let k = 0;
+  for (let i = 0; i < array.length - 1; i++) {
+    newArray[k] = array[i];
+    k++;
```

Comment:
What's the purpose of `k`? Why not just push directly into the array?

`newArray.push(array[i])`